elyra/pipeline/processor_kfp.py

```diff
@@ -169,6 +79,126 @@ def cc_pipeline():
 
         return None
 
+    def export(self, pipeline):
+        if pipeline.file_type not in ["tgz", "tar.gz", "zip", "yaml", "yml", "py"]:
```

Comment:
Although not necessarily germane to this PR, I think we're going to need to define a means of configuration that can be shared by the front.  I think adding a list-based config item (e.g., `extensions_extensions = ["tgz", "tar.gz", "zip", "yaml", "yml", "py"]`) that the front-end can also access (via the /api/config endpoint), would be good.  Since these are really a function of the platform (I'm assuming), perhaps a better name would be `kfp_extensions`, and `airflow_extensions` may have a different set.  We can talk about this another time.  For now, I think hardcoding the extensions is sufficient.